package.json

```diff
@@ -37,9 +38,10 @@
     "node": ">=0.4.x"
   },
   "dependencies": {
-    "underscore": "1.8.3",
     "mapnik-reference": "~6.0.2",
-    "optimist": "~0.6.0"
+    "optimist": "~0.6.0",
+    "torque-reference": "git+https://github.com/CartoDB/torque-reference.git",
```

Comment:
I added the github address because it wasn't published to npm yet, but happy to do so 👍